src/api/mobility.ts

```diff
@@ -22,6 +32,7 @@ export const getVehicles = (
     lon,
     range: Math.ceil(range),
     formFactors: FormFactor.Scooter, //TODO: Read from variables
```

Comment:
Is this something to cover on this PR? inject formFactor from outside?

Reply:
It's an old TODO. It may appear to be added now, since this code has been moved from two files to one. It will be resolved as soon as other vehicles than scooters are added to the app. (bikes and cars are _stations_)